testsuite/integration-arquillian/tests/base/testsuites/jdk-suite

```diff
@@ -0,0 +1,17 @@
+AccountRestServiceTest
```

Comment:
@mposolda can you take a look at this list and see if it's an okay list to test JDKs?

Reply:
For JDK, it can be good to test javascript providers as there were issues with them on the JDK 17. So package `org.keycloak.testsuite.script` might be good to add?

Also something related to Kerberos may be good as there are some differences between IBM JDK and others for example. Maybe `KerberosLdapTest`?

Are all admin REST API strictly required on different JDKs?

Reply:
Updated this one quite a bit, as it too much copy/paste from the database-suite. Could you take another look?